### PR TITLE
PD-201: document container runtimes

### PR DESCRIPTION
**Pull request by zoxpx**
* adding troubleshooting note about the K8s container runtimes, and since which px-version they got supported

### CODE REVIEW

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ description: Common errors
 
 ---
 
-* Failure to install Portworx:
+* Failure to install Portworx (selinux):
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]: Is this specifically for SELinux? I'd omit the parenthesis
```suggestion
* Failure to install Portworx on SELinux:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 2]
```diff
@@ -35,8 +35,9 @@ description: Common errors
       container process caused "process_linux.go:430: container init caused \
       \"write /proc/self/attr/keycreate: permission denied\"": unknown.
     ```
-    
-  * **EXPLANATION**:<br/> This error is a docker issue (see [moby#39109](https://github.com/moby/moby/issues/39109)), and it prevents running even the simplest containers:
+
+  * **EXPLANATION**:<br/> This error is a docker issue (see [moby#39109](https://github.com/moby/moby/issues/39109)),
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]:
```suggestion
  * **EXPLANATION**:<br/> This error is caused by a Docker issue ([moby#39109](https://github.com/moby/moby/issues/39109)),
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 3]
```diff
@@ -35,8 +35,9 @@ description: Common errors
...
+    and it prevents running even the simplest containers:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]: "it" in this case is Docker?
```suggestion
    which prevents Docker from running even the simplest containers:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 4]
```diff
@@ -45,5 +46,43 @@ description: Common errors
       \"write /proc/self/attr/keycreate: permission denied\"": unknown.
     ```
 
-  * to work around this issue, one should either turn off the [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) support, or make sure to use docker-package provided by the host's platform.
+  * to work around this issue, one should either turn off the [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux)
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]:
```suggestion
  * To work around this issue, either turn off [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux)
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 5]
```diff
@@ -45,5 +46,43 @@ description: Common errors
...
+    support, or make sure to use docker-package provided by the host's platform.
+<br/>
+
+---
+
+* Failure to install Portworx (kubernetes container runtime):
+  * You may have experienced the following issue deploying Portworx into Kubernetes:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]:
```suggestion
  * You may experience the following issue deploying Portworx into Kubernetes:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 6]
```diff
@@ -45,5 +46,43 @@ description: Common errors
...
     ```
 
-  * to work around this issue, one should either turn off the [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux) support, or make sure to use docker-package provided by the host's platform.
+  * to work around this issue, one should either turn off the [SELinux](https://en.wikipedia.org/wiki/Security-Enhanced_Linux)
+    support, or make sure to use docker-package provided by the host's platform.
+<br/>
+
+---
+
+* Failure to install Portworx (kubernetes container runtime):
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]: Similar to my other question. The parenthesis makes the statement vague. Is it "... the Portworx Kubernetes container runtime"? or "... Portworx using the Kubernetes container runtime"?

```suggestion
* Failure to install Portworx (kubernetes container runtime):
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 7]
```diff
@@ -45,5 +46,43 @@ description: Common errors
...
+    ...level=error msg="Could not talk to Docker/Containerd/CRI - please ensure \
+          '/var/run/docker.sock', '/run/containerd/containerd.sock' or          \
+          '/var/run/crio/crio.sock' are mounted"
+    ```
+
+  * **EXPLANATION**:<br/> In Kubernetes environments, Portworx installation starts by deploying OCI-Monitor Daemonset,
+    which monitors and manages the Portworx service.
+    In order to download, install and/or validate the Portworx service, the OCI-Monitor connects to the appropriate
+    Kubernetes container runtime via socket-files that need to be mounted into the OCI-Monitor's POD.
+  * Please inspect Portworx Daemonset spec, and ensure the appropriate socket-files/directories are mounted as volumes
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]:
```suggestion
  * Please inspect the Portworx Daemonset spec, and ensure the appropriate socket-files/directories are mounted as volumes
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 8]
```diff
@@ -45,5 +46,43 @@ description: Common errors
...
+    from host-system into the Portworx POD.
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]: Is there any reason to all-caps the word pod? 

```suggestion
    from the host-system into the Portworx pod.
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md [hunk 9]
```diff
@@ -45,5 +46,43 @@ description: Common errors
...
+    Alternatively you can [reinstall Portworx](/portworx-install-with-kubernetes/), or at minimum generate a new
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/troubleshooting/common-errors.md]:
```suggestion
    Alternatively, you can [reinstall Portworx](/portworx-install-with-kubernetes/), or at minimum generate a new
```

**Comment by zoxpx** [comment]
Hello @nathan-px -- thanks for your review.  I think I addressed issues you raised via follow-up 349e326 checkin.

**Comment by zoxpx** [comment]
Thanks Nathan - merging the changes.

**Pull request merged**